README.md

```diff
@@ -4,7 +4,9 @@
 
 Datadog Real User Monitoring (RUM) enables you to visualize and analyze the real-time performance and user journeys of your application’s individual users.
 
-The minimum supported version for the Datadog React Native SDK is React Native v0.63.4+. Compatibility with older versions is not guaranteed out-of-the-box.
+The minimum supported version for the RUM React Native SDK is React Native v0.63.4+. Compatibility with older versions is not guaranteed out-of-the-box.
+
+The RUM React Native SDK supports [Expo and Export Go][12]. For more information, see the [Expo documentation][13].
```

Comment:
Actually we don't support Expo Go (that would require a lot more work), but there are other ways to keep using Expo while using Datadog.

```suggestion
The RUM React Native SDK supports [Expo][12]. For more information, see the [Expo documentation][13].
```